docs/operators/kafkaInputOperator.md

```diff
@@ -280,3 +282,147 @@ Below is the configuration for “test” Kafka topic name and
 <value>localhost:2181</value>
 </property>
 ```
+
+
+### 0.9 Version of Kafka Input Operator
+
+### AbstractKafkaInputOperator
+
+This is the abstract implementation which is using 0.9 new consumer API that serves as base class
+for consuming messages from Kafka messaging system. This operator has key features like fault-tolerant, scalable, multi-cluster support, multi-topic support.
+
+#### Pre-requisites
+
+This operator referred the Kafka Consumer API of version
+0.9.0. So, this operator will work with any 0.9.x version of Apache Kafka.
```

Comment:
Rewrite:
This operator requires version 0.9.0 or later of the Kafka Consumer API.